### PR TITLE
Implement insight deployment frequency web api

### PR DESCRIPTION
**Pull request by Hosshii**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

rel #1142

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/app/server/grpcapi/web_api.go [hunk 1]
```diff
@@ -38,11 +39,14 @@ import (
 	"github.com/pipe-cd/pipecd/pkg/config"
 	"github.com/pipe-cd/pipecd/pkg/datastore"
 	"github.com/pipe-cd/pipecd/pkg/filestore"
+	"github.com/pipe-cd/pipecd/pkg/insight"
 	"github.com/pipe-cd/pipecd/pkg/insight/insightstore"
 	"github.com/pipe-cd/pipecd/pkg/model"
 	"github.com/pipe-cd/pipecd/pkg/rpc/rpcauth"
 )
 
+var deploymentFrequencyMinimumVersion = model.InsightDeploymentVersion_V0
```
Comment (nghialv) [inline pkg/app/server/grpcapi/web_api.go:48]:
```suggestion
const deploymentFrequencyMinimumVersion = model.InsightDeploymentVersion_V0
```

Comment (nghialv) [reply pkg/app/server/grpcapi/web_api.go:48]: And move this const into inside the `getDeploymentFrequency` function.

**Comment by Hosshii** [comment]
I fixed. PTAL

**Comment by knanao** [comment]
Nice catch!
/lgtm

**Comment by nghialv** [comment]
Thank you.
/approve

**Pull request merged**